platform/build.gradle

```diff
@@ -139,12 +139,13 @@ dependencies {
 
     api 'org.hibernate.validator:hibernate-validator:8.0.2.Final'
 
-    api 'org.hyperledger.besu:arithmetic:1.1.2'
-    api 'org.hyperledger.besu:blake2bf:1.1.2'
-    api 'org.hyperledger.besu:gnark:1.1.2'
-    api 'org.hyperledger.besu:ipa-multipoint:1.1.2'
-    api 'org.hyperledger.besu:secp256k1:1.1.2'
-    api 'org.hyperledger.besu:secp256r1:1.1.2'
+    api 'org.hyperledger.besu:besu-native-common:1.2.1'
+    api 'org.hyperledger.besu:arithmetic:1.2.1'
+    api 'org.hyperledger.besu:blake2bf:1.2.1'
+    api 'org.hyperledger.besu:gnark:1.2.1'
+    api 'org.hyperledger.besu:ipa-multipoint:1.2.1'
+    api 'org.hyperledger.besu:secp256k1:1.2.1'
+    api 'org.hyperledger.besu:secp256r1:1.2.1'
```

Comment:
this version bump could be a separate PR

Reply:
it could be, but the purpose of that version of besu-native was to enable this PR (forced native lib dependencies)